.github/workflows/integration-tests.yml

```diff
@@ -74,37 +80,17 @@ jobs:
           - el: ethrex
             zkvm: pico # See https://github.com/eth-act/ere/issues/174
           - el: ethrex
-            zkvm: zisk # See https://github.com/eth-act/ere/issues/XXX
-
-
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
-
-      - name: Install Rust toolchain
-        uses: dtolnay/rust-toolchain@nightly
-
-      - name: Install C toolchain dependencies
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y build-essential clang libclang-dev
-      
-      - name: Pull ere images
-        run: |
-          for variant in "base" "base-${{ matrix.zkvm }}" "cli-${{ matrix.zkvm }}"; do
-            src="ghcr.io/eth-act/ere/ere-${variant}:${ERE_TAG}"
-            dst="ere-${variant}:${ERE_TAG}"
-            docker pull "$src"
-            docker tag "$src" "$dst"
-          done
-
-      - name: Run benchmark
-        run: |
-          ${{ matrix.zkvm == 'openvm' && 'RAYON_NUM_THREADS=1' || '' }} RUST_LOG=warn,benchmark_runner=info ZKVM=${{ matrix.zkvm }} EL=${{ matrix.el }} cargo test --release -p integration-tests -- --test-threads=1 ${{ matrix.test }}
+            zkvm: zisk # See https://github.com/eth-act/ere/issues/186
+    uses: ./.github/workflows/run-benchmark.yml
```

Comment:
Most of deleted code in this file was extracted to this separate component for better modularity and avoid repetition.